cairo/tests/ethereum/cancun/vm/test_runtime.py

```diff
@@ -1,12 +1,34 @@
 from ethereum_types.bytes import Bytes
 from hypothesis import given
+from hypothesis import strategies as st
 
 from ethereum.cancun.vm.runtime import get_valid_jump_destinations
+from tests.utils.solidity import get_contract
+
+
+@st.composite
+def contract_code_strategy(draw) -> Bytes:
+    """Generate bytecode from actual Solidity contracts."""
+    # Get bytecode from our test contracts
+    counter = get_contract("Counter", "Counter")
+    erc20 = get_contract("ERC20", "KethToken")
+
+    # Choose between Counter, ERC20, or random bytes
+    is_contract = draw(st.booleans())
+
+    if is_contract:
+        # Choose between our contracts
+        contract = draw(st.sampled_from([counter, erc20]))
+        code = bytes(contract.bytecode_runtime)
+    else:
+        code = draw(st.from_type(Bytes))
+
+    return code
 
 
 class TestRuntime:
-    @given(code=...)
+    @given(code=contract_code_strategy())
```

Comment:
just use `@example` for adding particular test cases, see also https://github.com/kkrt-labs/keth/blob/8eb3bddfd586632f8fee5bd06c254c366730fc44/cairo/tests/src/utils/test_utils.py#L108